app/src/main/java/com/appunite/loudius/network/datasource/PullRequestsDataSourceImpl.kt

```diff
@@ -21,8 +21,6 @@ import com.appunite.loudius.network.model.RequestedReviewersResponse
 import com.appunite.loudius.network.model.Review
 import com.appunite.loudius.network.model.request.NotifyRequestBody
 import com.appunite.loudius.network.services.PullRequestsService
-import javax.inject.Inject
-import javax.inject.Singleton
```

Comment:
Question: Is it really building in the correct way? We have annotations below and here are deleted imports ;)